src/index.ts

```diff
@@ -59,42 +61,46 @@ app.use(morgan('tiny'))
 app.use('/api/v1/', router)
 const web3 = new Web3(argv['ethereum-node-url'] as string)
 
+const pool = workerpool.pool(path.join(__dirname, '/worker.js'), {
+  minWorkers: "max",
+  maxWorkers: 10
```

Comment:
TODO: What number of workers? Probably make it command line argument.

Reply:
yes, probably some power of two? Maybe 8 for now?

Reply:
I would default to `NUM_CPUS` and have it be a command line argument.

Reply:
```
const os = require('os')
const numCpus = os.cpus().length
```